mathics/builtin/numbers/constants.py

```diff
@@ -273,7 +273,7 @@ class Degree(_MPMathConstant, _NumpyConstant, _SympyConstant):
      = 0.0174532925199432957692369076849
     """
 
-    summary_text = "conversion factor from radians to sexagesimal degrees"
```

Comment:
This one is funny because you were the one that add this in a previous commit. I remember looking this up and spending time thinking about this the last PR. 

This is kind of why I find this distracting.

Reply:
I change it because your suggestion of using the text in Wolfram Reference

Reply:
What is funny here is that only appears the erased line, but not the new one.

Reply:
Yes, you added it in cbbcd279a

Reply:
Yes, I know, what I meant is in this PR I didn't erase it, but just changed the string, according to the text in the WR page.